docroot/themes/humsci/humsci_basic/sass-specs/sass-tests.scss

```diff
@@ -0,0 +1,11 @@
+// $current-theme needs to be set otherwise the tests will fail
+$default-theme: 'colorful';
+$current-theme: $default-theme;
+
+@import "true";
+@import "../src/scss/main.scss";
+
+// Import all test partials here
+@include describe('Sass Unit Tests') {
+  @import 'tools.mixins.themes';
+}
```

Comment:
Great setup! What do you think about moving the entire `sass-specs` folder into a tests directory? I'm thinking we could have JS tests in the future, and it would be nice to have them all organized together under one folder.

Reply:
good idea!